src/pages/home/sidebar/SidebarLinks.tsx

```diff
@@ -138,13 +135,13 @@ function SidebarLinks({onLinkClick, insets, optionListItems, isLoading, priority
      * @param {String} option.reportID
      */
     const showReportPage = useCallback(
-        (option) => {
+        (option: OptionData) => {
             // Prevent opening Report page when clicking LHN row quickly after clicking FAB icon
             // or when clicking the active LHN row on large screens
             // or when continuously clicking different LHNs, only apply to small screen
             // since getTopmostReportId always returns on other devices
             const reportActionID = Navigation.getTopmostReportActionId();
-            if (isCreateMenuOpen || (option.reportID === Navigation.getTopmostReportId() && !reportActionID) || (isSmallScreenWidth && isActiveReport(option.reportID) && !reportActionID)) {
+            if (isCreateMenuOpen ?? (option.reportID === Navigation.getTopmostReportId() && !reportActionID) ?? (isSmallScreenWidth && isActiveReport(option.reportID) && !reportActionID)) {
```

Comment:
you shouldn't use `??` in if statement use `||` if linter is complaining use `!!` operator for values which are type `boolean | undefined`